docs/schema/method-element-in-methods-bdcmetadata-schema.md

```diff
@@ -31,114 +34,30 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Child elements
 
-<table>
-<colgroup>
-<col width="50%" />
-<col width="50%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>Element</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The localized display names of the method.</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p><span sdata="link"><a href="properties-element-in-metadataobject-bdcmetadata-schema.md">Properties Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The properties of the method.</p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="accesscontrollist-element-bdcmetadata-schema.md">AccessControlList Element (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The access control list (ACL) of this method.</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p><span sdata="link"><a href="filterdescriptors-element-in-method-bdcmetadata-schema.md">FilterDescriptors Element in Method (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The filter descriptors of the method.</p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="parameters-element-in-method-bdcmetadata-schema.md">Parameters Element in Method (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The parameters of the method. A method cannot have more than one return parameter.</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p><span sdata="link"><a href="methodinstances-element-in-method-bdcmetadata-schema.md">MethodInstances Element in Method (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The method instances of the method.</p></td>
-</tr>
-</tbody>
-</table>
+|Element|Description|
+|-------|-----------|
+|[LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)](localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md)|The localized display names of the method.
+|[Properties Element in MetadataObject (BDCMetadata Schema)](properties-element-in-metadataobject-bdcmetadata-schema.md)|The properties of the method.
+|[AccessControlList Element (BDCMetadata Schema)](accesscontrollist-element-bdcmetadata-schema.md)|The access control list (ACL) of this method.
+|[FilterDescriptors Element in Method (BDCMetadata Schema)](filterdescriptors-element-in-method-bdcmetadata-schema.md)|The filter descriptors of the method.
+[Parameters Element in Method (BDCMetadata Schema)](parameters-element-in-method-bdcmetadata-schema.md)|The parameters of the method. A method cannot have more than one return parameter.
```

Comment:
Forgot beginning pipes on lines 53 and 54, and end pipes on lines 49 through 54.